torchtitan/config/job_config.py

```diff
@@ -239,6 +239,9 @@ class Training:
     deterministic: bool = False
     """Use deterministic algorithms wherever possible, may be slower"""
 
+    debug_moe_force_load_balance: bool = False
+    # if True, we force each experts get same amount of token via round-robin
```

Comment:
One last nit to let arg parser pick up the helper message.
```suggestion
    """If True, we force each experts to get the same amount of tokens via round-robin. This option is for debugging usage only."""
```